src/poetry/console/commands/init.py

```diff
@@ -141,9 +142,12 @@ def _init_pyproject(
 
         if is_interactive:
             question = self.create_question(
-                f"Version [<comment>{version}</comment>]: ", default=version
+                f"Version [<comment>{version}</comment>]: ",
```

Comment:
If package-mode is disabled, this now shows a version, which is not the default. We should probably set the version some lines above depending on the `disable-package-mode` option.